app/scripts/controllers/preferences.js

```diff
@@ -92,6 +92,7 @@ export default class PreferencesController {
         useNativeCurrencyAsPrimaryCurrency: true,
         hideZeroBalanceTokens: false,
         petnamesEnabled: true,
+        featureNotificationsEnabled: false,
```

Comment:
Similar to experimental petnames, we've placed the experimental feature notfications here in this preferences controller.